### ISSUE TITLE
Cannot login into admin anymore

### ISSUE THREAD

**Issue by Donderda** [shopware/platform#2099]
### PHP Version

8

### Shopware Version

6.4.5.0

### Expected behaviour

Successful login into admin.

### Actual behaviour

After updating to sw 6.4.5.0 I cannot login into the backend anymore. I get the error: 

`Mixed Content: The page at 'https://mysite.de/admin/#login' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://mysite.de/api/oauth/token'. This request has been blocked; the content must be served over HTTPS.` 

### How to reproduce

Use a reverse proxy to handle the SSL-Certificate and forward to the actual shop without SSL.

My `.env` has this line
```
SECURE_SCHEME=https
```

Before updating, it worked.

**Comment by shyim** [comment]
Configure https://symfony.com/doc/3.2/components/http_foundation/trusting_proxies.html correctly. `SECURE_SCHEME` should not have any affect on Shopware

**Comment by Donderda** [comment]
Is also set: 
```
TRUSTED_PROXIES=127.0.0.1,127.0.0.2,172.18.0.2,REMOTE_ADDR
```

**Comment by shyim** [comment]
do you have an x-forwarded-proto?

**Comment by Donderda** [comment]
yep: in my nginx config these values are set:
```
  proxy_set_header X-Real-IP $remote_addr;
  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_set_header X-Forwarded-Proto $scheme;
  proxy_set_header X-Forwarded-Port $server_port;
  proxy_set_header X-Forwarded-Host $host;
  proxy_set_header X-Forwarded-Server $host;
```

**Comment by shyim** [comment]
does your trusted proxies look like this? https://github.com/shopware/production/blob/6.4/public/index.php#L66

**Comment by Donderda** [comment]
When I add
```
var_dump(explode(',', $trustedProxies));
```

to this line,  get this output:
```
array(4) {
   [0]=>
   string(9) "127.0.0.1"
   [1]=>
   string(9) "127.0.0.2"
   [2]=>
   string(10) "172.18.0.2"
   [3]=>
   string(11) "REMOTE_ADDR"
}
```

**Comment by Donderda** [comment]
... and yes, the file is exactly the same :(

**Comment by Carrvin** [comment]
We have the same issue after updating one of our dev-instances to 6.4.5.0 (development-template). We are using Cloudfront as CDN with non-SSL traffic to the real ec2.

SECURE_SCHEME is not set in the env

Like it seems in the shopware 6 slack - channel, there are a lot people having trouble with this

**Comment by shyim** [comment]
Can you try?
https://github.com/shopware/platform/pull/2101

**Comment by Donderda** [comment]
Yep. That does the trick. Thank you :)

**Comment by Carrvin** [comment]
Works on our side as well. Is there a reason why this fix is not part of v6.4.5.1?

**Comment by arosenhagen** [comment]
*bump* Is there a reason why this fix is not part of v6.4.5.1?

**Comment by fgiering** [comment]
> 
> 
> Can you try? #2101

works!
